examples/hello_ocean.jl

```diff
@@ -0,0 +1,35 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (1, 64, 64),
+                            x = (0, 1), y = (-4, 4), z = (-4, 4),
+                            topology = (Periodic, Bounded, Bounded))
+
+model = IncompressibleModel(grid = grid,
+                            architecture = CPU(),
+                            advection = Oceananigans.Advection.WENO5(),
+                            tracers = :b,
+                            buoyancy = BuoyancyTracer(),
+                            closure = IsotropicDiffusivity(ν=1e-3, κ=1e-3))
+         
+@info "Simulating a rising buoyant bubble with" model
+
+set!(model, b = (x, y, z) -> exp(-y^2 - z^2))
+
+run!(Simulation(model, Δt=0.01, stop_iteration=800))
+
+# Analyze the data
+
+b = model.tracers.b
+
+plt = contourf(ynodes(b), znodes(b), interior(b)[1, :, :]',
```

Comment:
Why not an animation?
Or if we just plot, then I suggest we plot both the initial + final snapshot?

Reply:
animation is too much for "hello, ocean"... ?

Reply:
i think plotting the initial conditions would help. however, it would create 2 additional lines.

Reply:
hmm, if we make the plot (but don't save the data), and then just combine the plots after, it is just 2 additional lines. perhaps worth it.

Reply:
I'm definitely on board for more animation but a nice plot should be enough for the first introductory example (initial conditions would be pretty useful I think).

I think the bigger issue with doing animation is it would complicate the example since it either requires an output writer (probably a good thing to show although all the Literate.jl examples show how to use output writers). Without an output writer, the plotting has to happen in a loop that keeps extending `simulation.stop_time` which is not a nice pattern.